docs/source/how-to-downstream.mdx

```diff
@@ -5,19 +5,21 @@ stored on the Hub. You can use these functions independently or integrate them i
 own library, making it more convenient for your users to interact with the Hub. This
 guide will show you how to:
 
-* Specify a file to download from the Hub.
-* Download and cache a file on your disk.
+* Download and store a file from the Hub.
 * Download all the files in a repository.
 
-## Choose a file to download
+## Download and store a file from the Hub
 
-Use the `filename` parameter in the [`hf_hub_url`] function to retrieve the URL of a
-specific file to download:
+The [`hf_hub_download`] function is the main function for downloading files from the Hub.
+
+It downloads the remote file, stores it on disk (in a version-aware way), and returns its local file path.
+
+Use the `repo_id` and `filename` parameters to specify which file to download:
 
 ```python
->>> from huggingface_hub import hf_hub_url
->>> hf_hub_url(repo_id="lysandre/arxiv-nlp", filename="config.json")
-'https://huggingface.co/lysandre/arxiv-nlp/resolve/main/config.json'
+>>> from huggingface_hub import hf_hub_download
+>>> hf_hub_download(repo_id="lysandre/arxiv-nlp", filename="config.json")
+'/home/lysandre/.cache/huggingface/hub/bc0e8cc2f8271b322304e8bb84b3b7580701d53a335ab2d75da19c249e2eeebb.066dae6fdb1e2b8cce60c35cc0f78ed1451d9b341c78de19f3ad469d10a8cbb1'
```

Comment:
Happy to see my cache displayed here :smiley: this should likely be anonymized at some point across the repo